multiqc/modules/whatshap/whatshap.py

```diff
@@ -0,0 +1,321 @@
+                (
+                    "heterozygous_snvs",
+                    {
+                        "id": "heterozygous_snvs",
+                        "title": "Heterozygous SNVs",
+                        "description": "The number of biallelic, heterozygous SNVs in the "
+                        "input VCF. This is a subset of Heterozygous "
+                        "Variants.",
+                        "hidden": True,
+                    },
+                ),
+                (
+                    "unphased",
+                    {
+                        "id": "unphased",
+                        "title": "Unphased Variants",
+                        "description": "The number of biallelic, heterozygous variants that "
+                        "are not marked as phased in the input VCF. This "
+                        "is a subset of heterozygous_variants.",
+                        "hidden": False,
+                    },
+                ),
+                (
+                    "phased",
+                    {
+                        "id": "phased",
+                        "title": "Phased Variants",
+                        "description": "The number of biallelic, heterozygous variants that "
+                        "are marked as phased in the input VCF. This is "
+                        "a subset of heterozygous_variants. Also, phased + "
+                        "unphased + singletons = heterozygous_variants.",
+                        "hidden": False,
+                    },
+                ),
+                (
+                    "phased_snvs",
+                    {
+                        "id": "phased_snvs",
+                        "title": "Phased SNVs",
+                        "description": "The number of biallelic, heterozygous SNVs that are "
+                        "marked as phased in the input VCF. This is a subset "
+                        "of phased.",
+                        "hidden": True,
+                    },
+                ),
+                (
+                    "blocks",
+                    {
+                        "id": "blocks",
+                        "title": "Blocks",
+                        "description": "The total number of phase sets/blocks.",
+                        "hidden": True,
+                    },
+                ),
+                (
+                    "singletons",
+                    {
+                        "id": "singletons",
+                        "title": "Singletons",
+                        "description": "The number of blocks that contain exactly one " "variant.",
+                        "hidden": True,
+                    },
+                ),
+                (
+                    "bp_per_block_sum",
+                    {
+                        "id": "bp_per_block_sum",
+                        "title": "Total Phased bp",
+                        "description": "Description of the distribution of non-singleton "
```

Comment:
That sentence (copied from the WhatsHap docs) was intended to refer to all the `bp_per_block_*` values. Since here the description now only refers to the `bp_per_block_sum` value, this should be: "The sum of the lengths of all non-singleton blocks, where the length of a block ..."

(It’s formatted a bit strangely in the docs; I’ll need to fix that.)

Reply:
I've updated the description, thanks.